backend/src/layer/data/swka_parser/html_parser.rs

```diff
@@ -369,20 +380,132 @@ impl HTMLParser {
             .collect()
     }
 
-    fn get_dish_type(dish_node: &ElementRef) -> Option<MealType> {
+    fn get_dish_type(dish_node: &ElementRef) -> Option<FoodType> {
         let dish_type_node = dish_node.select(&DISH_TYPE_NODE_CLASS_SELECTOR).next()?;
         dish_type_node
             .value()
             .attr(DISH_TYPE_ATTRIBUTE_NAME)
-            .map(MealType::parse)
+            .map(FoodType::parse)
     }
 
-    fn get_dish_env_score(dish_node: &ElementRef) -> Option<u32> {
-        let env_score_node = dish_node.select(&ENV_SCORE_NODE_CLASS_SELECTOR).next()?;
-        env_score_node
-            .value()
-            .attr(ENV_SCORE_ATTRIBUTE_NAME)?
-            .parse::<u32>()
+    fn get_dish_env_score(dish_node: &ElementRef) -> Option<EnvironmentInfo> {
+        let env_info = EnvironmentInfo {
+            average_rating: Self::get_average_env_score(dish_node)?,
+            co2_rating: Self::get_co2_rating(dish_node)?,
+            co2_value: Self::get_co2_value(dish_node)?,
+            water_rating: Self::get_rating(dish_node, "div.wasser_bewertung")?,
```

Comment:
should these selectors (`div.wasser_bewertung' etc) not rather be constants?

Reply:
maybe that would make thinks more unreadable though, just an idea...

Reply:
Yes. This is more quick and dirty. I can change it tomorrow

Reply:
I actually would not just create global constants for constants' sake, especially if those are only used once. That way, you always have to scroll from the usage to the top of the document to find out what the constant is referring to. If it is self-explanatory, I would just put it as a "magic string" where it is used, otherwise maybe declare a constant/variable next to where it is used.